### ISSUE TITLE
Meka-Tool improvements

### ISSUE THREAD

**Issue by 8Times** [mekanism/Mekanism-Feature-Requests#254]
I'm currently using Mekanims version 10.0.21.448.

In this version there are multiple things I really wish for:

1. A general current FE consumption per use (attack and digging) on the tooltip or HUD in the left corner.
2. Remove the unchangeable "Shift + Scrollwheel" to change Efficiency or include it in the settings. (really messes with me when I'm crouching and want to change my item)
3. A button to enable and disable Extended Vein Mining.
4. That Extended Vein Mining in combination with full efficiency sucks the Meka-Tool from all energy because of the insane energy consumption it has. (Is this a wanted?)
5. An upgrade card which gives excavation area from 3x3, 5x5 to 7x7 block area.
6. An upgrade card which increases the reach of the tool.

**Comment by 8Times** [comment]
> 
> 
>     1. [mekanism/Mekanism#7069](https://github.com/mekanism/Mekanism/issues/7069)
> 
>     2. I will fix one thing slightly regarding it in that you can disable it changing mode in the module tweaker, but it doesn't currently let it scroll past it (you can disable the shift + scrollwheel in general for all our items that support it in config/Mekanism/client.toml)
> 
>     3. Not happening until we have a way to have multiple mode changes per item though it is something we are aware of
> 
>     4. [mekanism/Mekanism#6786](https://github.com/mekanism/Mekanism/issues/6786)
>        5 and 6. I will leave this issue open for them (not sure if/when they would happen) but I am going to transfer this issue to the feature request repo.

I'd say mekanism/Mekanism#7069 doesn't fit because I want particurarely the energy usage not how full it is.

**Comment by pupnewfster** [comment]
Ah I misread your thing given I saw "FE" and "HUD" and just presumed. I will consider adding some way to represent it as well at some point, though given how many different types of operation there are not sure how simple it will be to do. Maybe some system could be made at some point that displays recent energy costs though it would need a pretty decent rewrite of how the energy is handled currently for it so not sure.

**Comment by orchidalloy** [comment]
As it stands the Atomic Disassembler is just more useful than the Mekatool, which is unacceptable. Give the Mekatool the same circular GUI to let you choose settings to turn on and off.

**Comment by pupnewfster** [comment]
From our FAQ:
**Q:** *Why can't I quickly change the Meka-Tool's vein mining mode?*
**A:** We currently only can have one mode change system setup and we have decided that the most important/impactful thing to allow changing is the mining speed. This is not up for debate and will stay as such until we expand the system to know how to handle multiple types of modes

**Comment by orchidalloy** [comment]
That's false, you have a different mode change in the form of a wheel for the atomic dissassembler.

**Comment by pupnewfster** [comment]
No. As far as the atomic disassembler is concerned they are all "speed" modes.
